src/eventcontroller/DefaultEventController.ts

```diff
@@ -96,7 +134,36 @@ export default class DefaultEventController implements EventController {
       await this.eventReporter?.reportEvent(timestampMs, name, flattenedAttributes);
```

Comment:
This would not work since the variable is now changed to `this._eventReporter`, so you have to update the usage accordingly.

Reply:
This works cause there's a getter for the EventReporter called eventReporter.